test/src/lib.rs

```diff
@@ -293,10 +317,27 @@ mod test_math {
         });
     }
 
-    // TODO: add more tests
     #[test]
     fn test_pt() {
-        // assert_eq!(pt(0.1, 1.0, false, false), unsafe { c::pt(0.1, 1.0, 0, 0) });
+        assert_eq!(pt(0.1, 1.0, false, false), unsafe {
+            c::pt(0.1, 1.0, false, false)
+        });
+        assert_eq!(pt(-0.1, 1.0, false, false), unsafe {
+            c::pt(-0.1, 1.0, false, false)
+        });
+        assert_eq!(pt(1.0, 1.0, false, false), unsafe {
+            c::pt(1.0, 1.0, false, false)
+        });
+        assert!(abs_diff_eq!(
+            pt(1.0, 1.0, false, true),
+            unsafe { c::pt(1.0, 1.0, false, true) },
+            epsilon = 1e-15
+        ));
+        assert!(abs_diff_eq!(
+            pt(1.0, 1.0, true, true),
+            unsafe { c::pt(1.0, 1.0, true, true) },
+            epsilon = 1e-15
+        ));
```

Comment:
Nit: also one test where the second arg is not 1.0?

Reply:
Actually you've found an error in my conversion.
Thanks for the nitpicking :) saved a lot of future work.

I added a `*n` in one of the `pbeta`s...